Lesson21/bilge/todo-app/script.js

```diff
@@ -0,0 +1,47 @@
+ // Task 1: Every todo list item should have a delete button attached to it. When delete button is clicked, corresponding list item should be removed from the list.
+  // Task 2: When a user clicks on the text of the todo item, only the text should be crossed (add line-through text decoration)
+  // Task 3: A user should only be able to add an actual todo item to the list (do not allow adding empty string or similar)
+  // Task 4: Clear the input field after the todo list items is added.
+
+  document
+  .getElementById("todoForm")
+  .addEventListener("submit", function (event) {
+    event.preventDefault();
+
+    const todoInputElement = document.getElementById("todoInput");
+    const toDoListElement = document.getElementById("toDoList"); // ul
+
+    // Task 3:
+    const newItem = todoInputElement.value.trim();
+    if (newItem === '') {
+      alert('There is nothing to add to the list.');
```

Comment:
Instead of using alert function you can create an initially empty text element on the page, if error happens, add the text to it to inform the user about what went wrong.